test/plugin/test_out_google_cloud.rb

```diff
@@ -346,6 +346,13 @@ def test_statusz_endpoint
     end
   end
 
+  def test_monitoring_config
```

Comment:
Why are you only adding this on the non-gRPC path? Shouldn't this be in `base_test.rb` instead?

Reply:
Ah, because I didn't understand how these tests are organized. I assumed changes to out_google_cloud.rb were tested in test_out_google_cloub.rb.

Moved to base_test.rb.